samples/cpp/3calibration.cpp

```diff
@@ -351,8 +351,8 @@ int main( int argc, char** argv )
         for( k = 0; k < small_canvas.rows; k += 16 )
             line(small_canvas, Point(0, k), Point(small_canvas.cols, k), Scalar(0,255,0), 1);
         imshow("rectified", small_canvas);
-        int c = waitKey(0);
-        if( c == 27 || c == 'q' || c == 'Q' )
+        int c = waitKey(0) & 0xFF;
+        if( c == 27 || c == ord('q') || c == ord('Q') )
```

Comment:
Hi berak. Discovered that C indeed does not have the ord function. As to the conversion. I read somewhere that 'char' gives problems when placed in an into when values larger then 127 occur. Not sure why though!